src/uwtools/tests/test_rocoto.py

```diff
@@ -52,3 +53,14 @@ def test_write_rocoto_xml(tmp_path):
 
     expected = support.fixture_path("hello_workflow.xml")
     support.compare_files(expected, output)
+
+
+def test_rocoto_xml_is_valid():
+    input_xml = support.fixture_path("hello_workflow.xml")
+    tree = etree.parse(input_xml)
+    root = etree.tostring(tree.getroot())
+
+    breakpoint()
+    with resources.as_file(resources.files("uwtools.resources")) as resc:
+        relaxng_doc = str(resc / "schema_with_metatasks")
+    relaxng = etree.RelaxNG(file=relaxng_doc)
```

Comment:
Getting an error message: 
`E   lxml.etree.RelaxNGParseError: unknown error
src/lxml/relaxng.pxi:100: RelaxNGParseError`

When originally passing the `relaxng_doc` I was getting the file path back as a PosixPath object and converted it to a string to pass to the RelaxNG class, where used this [documentation](https://lxml.de/api/lxml.etree.RelaxNG-class.html) to confirm using a keyword `file` to pass the path in. Any suggestions on working with this error?

I pulled `schema_with_metatasks.rng` from Chris' rocoto repo as well.

Reply:
The file is called `schema_with_metatasks.rng` -- the code above doesn't have the extension. See if that helps.

This works for me:
```
>>> from lxml import etree
>>> with open("schema_with_metatasks.rng", "r") as f:
...     schema = etree.RelaxNG(etree.parse(f))
... 
>>> 
```
That's a slightly different method that you're using -- you still need to use `resources.as_file()` to get the resources directory and then construct the filename, but maybe the issue was just the missing extension.

Reply:
That worked, thank you!